src/main/content/_assets/js/blog.js

```diff
@@ -93,8 +96,13 @@ var blog = function(){
                 excludeList.push(tagList[i].tag.toLowerCase());
                 excludeStr = excludeStr + tagList[i].tag.replace("_", " ") + ", ";
             } else {
-                includeStr = includeStr + ("." + tagList[i].tag.toLowerCase());
-                filterStr = filterStr + tagList[i].tag.replace("_", " ") + ", ";
+                if(translations[tagList[i].tag.replace("_", " ").toLowerCase()]){
+                    includeStr = includeStr + ("." + tagList[i].tag.toLowerCase());
```

Comment:
includeStr is the same on line 100 and 103 it could be the first line inside the else before line 99, but not a big deal